### ISSUE TITLE
Training Time

### ISSUE THREAD

**Issue by cantabile-kwok** [jaywalnut310/vits#44]
How much time do you need for training to get the performance in your paper and web demo? I recently started to do experiments on LJspeech and found the training really slow (10+ days for 800k steps).

**Comment by nikich340** [comment]
Yes, it's quite slow, but results seems to be really the best from all open-source tts frameworks.

**Comment by heya5** [comment]
hello, what kind of GPU devices did you use to train **800k steps with 10+ days** ?

**Comment by BinLiang2021** [comment]
Hello! I have the same question about what kind of GPU you used for this experiment? Thanks a lot. Hope you are well.

**Comment by cantabile-kwok** [comment]
@BinLiang2021 @heya5 I was using 2080ti for that, and I also want to know the normal speed for other people on other machines.
